hack/release.sh

```diff
@@ -22,12 +22,12 @@ source $(dirname $0)/../vendor/knative.dev/test-infra/scripts/release.sh
 # Yaml files to generate, and the source config dir for them.
 declare -A COMPONENTS
 COMPONENTS=(
-  ["eventing-core.yaml"]="config/core"
-  ["eventing-crds.yaml"]="config/core/resources"
-  ["channel-broker.yaml"]="config/brokers/channel-broker"
-  ["mt-channel-broker.yaml"]="config/brokers/mt-channel-broker"
-  ["in-memory-channel.yaml"]="config/channels/in-memory-channel"
-  ["upgrade-to-v0.14.0.yaml"]="config/upgrade/v0.14.0"
+  ["eventing-core.yaml"]="-R -f config/core"
```

Comment:
I do not like that declaration here.  I liked it before better

how about doing some more explicit `ko resolve` runs, like serving:

https://github.com/knative/serving/blob/master/hack/generate-yamls.sh#L86-L104

Reply:
/hold